.github/workflows/hydroshare-resouce-to-beta.yml

```diff
@@ -0,0 +1,62 @@
+name: Prod to Beta
+
+on:
+  workflow_dispatch:
+    inputs:
+      resource_ids:
+        description: 'Comma-separated list of resource IDs to check and sync'
+        required: true
+
+jobs:
+  check-and-sync:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Install MinIO CLI (mc)
+      run: |
+        curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
+        chmod +x mc
+        sudo mv mc /usr/local/bin/mc
+
+    - name: Set Up MinIO Environment Variables
+      env:
+        MC_HOST_prod: https://${{ secrets.MINIO_PROD_ACCESS_KEY }}:${{ secrets.MINIO_PROD_SECRET_KEY }}@${{ secrets.MINIO_PROD_URL }}
+        MC_HOST_beta: https://${{ secrets.MINIO_BETA_ACCESS_KEY }}:${{ secrets.MINIO_BETA_SECRET_KEY }}@${{ secrets.MINIO_BETA_URL }}
```

Comment:
Maybe drop this step and put the env assignments with the others in the step below?